internal/mockeec/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"net/http"
+)
+
+func main() {
+	port := flag.String("p", "8000", "port to listen on")
+	dir := flag.String("d", ".", "directory to serve files from")
```

Comment:
does it make sense to make the mockeec at least a little bit configurable? Maybe for future usage in a different component

Reply:
What other options do you think would be helpful? My goal here is essentially just to have this be a basic fileserver for quick demonstration with a user-run Collector, particularly to see reloads work in action.

Reply:
For example reading the port and directory from command line parameters, nothing serious here. It can cause problems for somebody running it locally, since port 8000 can be already in use.

Reply:
This is a command line parameter and not a static value.

Reply:
ah sorry I misread the code. Ignore my comments please